content/fr/community/_index.md

```diff
@@ -123,29 +123,29 @@ title: "Community"
   <div class="container">
     <div class="row justify-content-center">
       <div class="col-12 text-center">
-        <p class="section-title mb-5 mt-3 h1">How We Work</p>
-        <p>Affiliation with the InnerSource Commons is tied to individuals, not companies. We've included some guidance on how we work below. No fees are required to participate. Content is made available under the <a href="http://creativecommons.org/licenses/by-sa/4.0/">Creative Commons Attribution-ShareAlike 4.0 International License</a>.</p>
+        <p class="section-title mb-5 mt-3 h1">Comment nous travaillons</p>
+        <p>L'affiliation à l'ISC est liée aux individus, et non aux entreprises. Vous trouverez ci-dessous quelques détails sur notre mode de fonctionnement. Aucun frais n'est exigé pour participer. Le contenu est mis à disposition sous la Licence <a href="http://creativecommons.org/licenses/by-sa/4.0/">Creative Commons Attribution-ShareAlike 4.0 International License</a>.</p>
       </div>
       <div class="col-md-4 col-sm-6 mb-4">
         <div class="feature-card text-center bg-light">
           <i class="ti-bookmark-alt mb-3"></i>
-          <h4 class="mb-2">Tools & Resources</h4>
-          <p>Most of our discussions happen in Slack. We have public and private projects on <a href="http://github.com/InnerSourceCommons">GitHub</a> and store a lot of our working documents in gDrive.
+          <h4 class="mb-2">Outils & Ressources</h4>
+          <p>La plupart de nos discussions se déroulent sur Slack. Nous avons des projets publics et privés sur <a href="http://github.com/InnerSourceCommons">GitHub</a> et nous stockons beaucoup de nos documents de travail dans gDrive.
           </p>
         </div>
       </div>
       <div class="col-md-4 col-sm-6 mb-4">
         <div class="feature-card text-center bg-light">
           <i class="ti-comments mb-3"></i>
           <h4 class="mb-2">Chatham House Rule</h4>
-          <p>All discussions in our Slack channel are held under the <a href="https://www.chathamhouse.org/about-us/chatham-house-rule">Chatham House Rule</a> to create a safe space where people can ask real questions.</p>
+          <p>Toutes les discussions dans notre canal Slack se déroulent sous la règle dite du <a href="https://www.chathamhouse.org/about-us/chatham-house-rule">Chatham House Rule</a> pour créer un espace sûr où les gens peuvent poser toutes leurs questions.</p>
         </div>
       </div>
       <div class="col-md-4 col-sm-6 mb-4">
         <div class="feature-card text-center bg-light">
           <i class="ti-face-smile mb-3"></i>
-          <h4 class="mb-2">Code of Conduct</h4>
-          <p>Be nice and kind. If you would like more details, please read our <a href="/about/codeofconduct/">Code Of Conduct</a>.</p>
+          <h4 class="mb-2">Code de Conduite</h4>
+          <p>Soyez gentil et poli. Si vous souhaitez obtenir plus de détails, veuillez lire notre <a href="/about/codeofconduct/">Code de Conduite</a>.</p>
```

Comment:
Suggestion : Soyez bienveillant et courtois

Reply:
Sure :) this is much better, taken into account in next commit.